### PR TITLE
add more fe e2e test cases for online debug

### LINKED ISSUE #1291: test(fe): add e2e test for online debug
# Feature request

## Please describe your feature

as mentioned in https://github.com/apache/apisix-dashboard/pull/1287#pullrequestreview-567116648 and https://github.com/apache/apisix-dashboard/pull/1239#issuecomment-755963097, we need to add e2e test cases to online debug.

### PR DESCRIPTION
**Pull request by liuxiran**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
add more test cases to fix #1291 
___
### Bugfix
- Description
fix the related bug during testing
- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Please add the corresponding test cases if necessary.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

File: web/cypress/fixtures/route-json-data.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      }],
+      "pass_host": "pass",
+      "timeout": {
+        "connect": 6,
+        "send": 6,
+        "read": 6
+      }
+    }
+  }
+}
```
Comment (LiteSun) [inline web/cypress/fixtures/route-json-data.json:30]: new line please 😁

Comment (liuxiran) [reply web/cypress/fixtures/route-json-data.json]: done

**Comment by liuxiran** [comment]
https://github.com/apache/apisix-dashboard/pull/1612/checks?check_run_id=2146684625
just waiting for #1619 to solve this ci error

**Comment by liuxiran** [comment]
also cc @guoqqqi to review when you have time, thanks a lot

**Review by bzp2010 — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Comment by juzhiyuan** [comment]
still failed https://github.com/apache/apisix-dashboard/pull/1612/checks?check_run_id=2146684625

**Comment by liuxiran** [comment]
> still failed https://github.com/apache/apisix-dashboard/pull/1612/checks?check_run_id=2146684625

yep, just waiting for #1619, which has another fe e2e test failed, need some time to take a look

**Review by juzhiyuan — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**